components/elm/cime_config/testdefs/testmods_dirs/elm/polygonal_tundra/user_nl_elm

```diff
@@ -0,0 +1,3 @@
+use_polygonal_tundra = .true.
+use_arctic_init = .true.
+fsurdat = '$DIN_LOC_ROOT/lnd/clm2/surfdata_map/surfdata_1x1pt_IcyCape_polygons_c240909.nc'
```

Comment:
Similarly, if the `-res` for the test is changed, this `./xmlchange` won't be needed.

Reply:
Removed the fsurdat line